### PR TITLE
ci(securityscoreboard): add workflow to update scoreboard on push and add badge

### PR DESCRIPTION
**Pull request by secustor**
## Changes
Add the security scoreboard badge and adds a workflow to update it regurarly 

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: .github/workflows/scorecard.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Scorecard supply-chain security
+on:
+  schedule:
+    - cron: '0 5 * * 1'
+  push:
+    branches: [ "main" ]
```
Comment (viceice) [inline .github/workflows/scorecard.yml:6]:
```suggestion
    branches:
      - main
```
I prefer the list syntax, less conflict potential

File: .github/workflows/scorecard.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+permissions: read-all
```
Comment (viceice) [inline .github/workflows/scorecard.yml:8]: do we really need read all?

**Comment by viceice** [comment]
needs prettier fix

File: .github/workflows/scorecard.yml (hunk 2 above)
Comment (secustor) [reply .github/workflows/scorecard.yml:9]: That's based on their documentation

**Review by viceice — DISMISSED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**